src/ui/Analytics/Analytics.ts

```diff
@@ -254,6 +257,8 @@ export class Analytics extends Component {
       let event = this.componentOptionsModel.getEventName(Model.eventTypes.changeOne + ComponentOptionsModel.attributesEnum.searchHub);
       this.bind.onRootElement(event, (args: IAttributeChangedEventArg) => this.handleSearchHubChanged(args));
     }
+
+    this.createClientId();
```

Comment:
The `clientId` is initialized here since `Analytics` are inherently going to be initialized when a user is intended to be tracked.